developers_chamber/bin/pydev.py

```diff
@@ -30,28 +33,7 @@
         import scripts
 
 
-LOGGING = {
-    'version': 1,
-    'formatters': {
-        'standard': {
-            'format': '%(asctime)s [%(levelname)s]: %(message)s',
-        },
-    },
-    'handlers': {
-        'default': {
-            'class': 'logging.StreamHandler',
-            'formatter': 'standard',
-        },
-    },
-    'loggers': {
-        '': {
-            'handlers': ('default',),
-            'level': 'INFO',
-        },
-    },
-}
-logging.config.dictConfig(LOGGING)
-
+coloredlogs.install(milliseconds=True)
```

Comment:
This outputs nice, colored and formatted logs:
https://camo.githubusercontent.com/051f32c9866c6f552682955b5fde580778c63a94/68747470733a2f2f636f6c6f7265646c6f67732e72656164746865646f63732e696f2f656e2f6c61746573742f5f696d616765732f64656661756c74732e706e67

Reply:
wow it looks nice good job